kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaServiceReconciler.java

```diff
@@ -54,14 +51,8 @@ public final class KafkaServiceReconciler implements
 
     public static final String SECRETS_EVENT_SOURCE_NAME = "secrets";
     public static final String CONFIG_MAPS_EVENT_SOURCE_NAME = "configmaps";
-
-    public static AnyLocalRef asRef(TrustAnchorRef reference) {
-        AnyLocalRef result = new AnyLocalRef();
-        result.setGroup(reference.getGroup());
-        result.setKind(reference.getKind());
-        result.setName(reference.getName());
-        return result;
```

Comment:
This is exactly the mapping code which would avoid the tricky equality problem in the `TrustAnchorRef`.